library/Cube/Web/Controller.php

```diff
@@ -148,6 +147,11 @@ protected function renderCube(): void
 
         $this->view->url = Url::fromRequest()->onlyWith($this->preserveParams);
         $viewUrlParams = $this->view->url->getParams()->toArray(false);
+
+        if (! empty($viewUrlParams['dimensions'])) {
+            $viewUrlParams['dimensions'] = urldecode($viewUrlParams['dimensions']);
+        }
```

Comment:
What about the other view url params? I think it makes sense to decode all params and not just `dimensions`. Please also use `rawurldecode()` instead.